docs/reference/r-releases/r-rev185_149.md

```diff
@@ -0,0 +1,96 @@
+# Revision 185/149
+<sub>???, Oct ??, 2024</sub>
+
+Dear community,
+
+??? 
+
+Please reach out should you have any question, comment, feedback or information. You can find us here in [Matrix](https://matrix.to/#/#charmhub-data-platform:ubuntu.com) or also on [Discourse](https://discourse.charmhub.io/).
+
+## Kafka
+
+### Features
+* [DPE-2285] Refer to Charmhub space from GitHub (#200)
```

Comment:
I would leave it as is, as it is the name of the PR and it would be more confusing otherwise.

Reply:
We talked to @a-velasco and as far as I understood, she pays a lot of attention to rephrasing entries to release notes to make them more meaningful. 
I don't see a big problem with these entries being renamed as long as the link to the PR is there. But if you think it's confusing, can't we just rename the PR accordingly?

Reply:
@deusebio For some context, since we don't have separate release notes for high-level users of the charm and for developers, I try to make them useful for both. This means rephrasing some of the PR titles so that someone can understand what was done just by reading that sentence, rather than having to do a deep dive into GitHub/Jira to figure out if it's relevant to them or not.

I think this extra effort could be mitigated in two ways:
1. Encouraging more meaningful PR titles (admittedly not so easy to do, but we would provide guidance and examples)
2. Separating user release notes from developer release notes. Dev release notes can be easily auto-generated from the commits; the question is just where to put them (e.g. CHANGELOG.md, Discourse..?). User release notes could remain on Discourse and focus on a just few major highlights for that release with a bit of explanation, links to docs, etc.

Reply:
As I was saying to @izmalk, right now there is a nice correspondence between Release notes, PR Titles and commits. We can rename PRs but we can't rename commits. Also sometimes (when using rebase and merge) there may exists commits without the PR number. That's why I would find confusing to break this correspondence. 

Anyhow, my point of view is that rather than changing the release notes, we should nudge ourself to have meaningful PR titles. I believe we are already aiming at that (I don't think that there are engineers purposely putting misleading titles or descriptions), but what is meaningful is somewhat subjective. Providing guidance could definitely help to improve this. So, overall, I'd rather fix that over changing the names.

I'm not against having high-level user release notes, but this is probably for product managers to craft and have a call on.